### ISSUE TITLE
PHP 8 bug on Debian

### ISSUE THREAD

**Issue by williamdes** [zetacomponents/ConsoleTools#13]
Using the [Debian package](https://tracker.debian.org/pkg/php-zeta-console-tools) i had this error.
```
PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /usr/share/php/ezc/ConsoleTools/input.php on line 535
```

**Comment by williamdes** [comment]
cc @DavidPrevot

**Comment by derickr** [comment]
This should already be fixed in the 1.7.1 release, please upgrade. 

https://github.com/zetacomponents/ConsoleTools/commit/8b9a8b149775245e5341e1f835006df8efb746f1

There are however other warnings for PHP 8, but that's not related to that error.
